doc/installation.txt

```diff
@@ -53,9 +53,22 @@ Since Bazaar is mostly written in Python, just installing the main Bazaar
 package is sufficient. For Git, you need to install the GitPython_ package.
 For Mercurial, you need to install the hgapi_ package.
 
-.. _python.org: http://www.python.org
+
+Command completion for bash
+---------------------------
+
+Sumatra comes with a limited bash completion facility.
+You can install it to you system by sourcing the file
+`smt-completion.sh` in your `.bashrc` or `.profile.
+By default, Sumatra installs this script to your
+`/usr/bin` directory by default but moving it
+elsewhere (e.g. to `~/.bash_completion.d/`)
+is recommended.
+
+
+.. _python.org: https://www.python.org
 .. _Django: https://www.djangoproject.com
-.. _Mercurial: http://mercurial.selenic.com/
+.. _Mercurial: https://mercurial.selenic.com/
 .. _Subversion: http://subversion.tigris.org/
 .. _Git: http://git-scm.com/
```

Comment:
The issue is not whether it supports `https`, but whether `http` redirects to `https`. The Mercurial website does this, the Git website does not.

(these changes came from running `make linkcheck` in the `doc` directory, and trying to eliminate unnecessary redirects).